packages/react-conformance-griffel/src/overridesWin.ts

```diff
@@ -50,8 +50,10 @@ export const overridesWin: ConformanceTest = (componentInfo, testInfo) => {
     document.body.appendChild(container);
   });
 
-  afterEach(() => {
+  afterEach(async () => {
     if (container) {
+      const ReactDOM = await import('react-dom');
+      ReactDOM.unmountComponentAtNode(container);
```

Comment:
After removing the "nuclear" `document.body.innerHTML = ''` cleanup in react-conformance, the incomplete cleanup here was causing subsequent tests to fail if the component rendered anything outside of the container node (such as PopoverSurface rendering a Portal).